### PR TITLE
New structure for the app

### PR DESCRIPTION
**Pull request by PetNamedEr**
First version with folder structure

### CODE REVIEW

File: models/survey.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        self.description = description_
+
+    def __repr__(self):
+        return "<Question: {}>".format(self.name)
+
+class surveyForm(Form):
+    id = IntegerField("ID")
+    description = StringField("Description")
+
+@app.route('/addNewSurvey', methods=['GET', 'POST'])
```
Comment (Arvinje) [inline models/survey.py]: These routes and their underlying methods deserve a directory of their own. Try to move all routes and their functions to `/controllers` path. I believe these are just for testing cause they do **NOT** follow the specs stated on Trello. Take a look at "Create CRUD for Survey" [here](https://trello.com/c/gJl3omU7).

Comment (PetNamedEr) [reply models/survey.py]: Yes, they are just a test for IO :)! I'll try it out with controllers. Is there some common practice on naming controller-classes and files?

Comment (Arvinje) [reply models/survey.py]: @PetNamedEr I guessed 😉 Well, controllers should be named as plural nouns. Like "Surveys" or "Questions". Though models on the other hand should be named as singular nouns.

**Comment by Arvinje** [comment]
[Set up dev environment](https://trello.com/c/G7Qlggq7/28-set-up-dev-environment)

**Comment by Arvinje** [comment]
[Create "Survey" model](https://trello.com/c/D7Lmdi8I/22-create-survey-model)

File: main.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
+from flask import render_template, request
+
+import models.survey
+
+from app import app, db
+
```
Comment (InkaSimola) [inline main.py]: Maybe all of the routing should be elsewhere? Something like this:

main controllers

And main could then only contain someting along these lines:
```
if __name__ == '__main__':
        app.debug = True
        app.run(host = '0.0.0.0', port=5000)
```

File: models/survey.py (hunk 1 above)
Comment (PetNamedEr) [reply models/survey.py]: I made controllers/surveys.py

File: main.py (hunk 2 above)
Comment (PetNamedEr) [reply main.py]: For me all this is OK - it was my first Flask-attempt based on 2-3 tutorials :D. There must be a lot of better ways to do some of the stuff - if so, let me know and I'll change it 8)

Comment (PetNamedEr) [reply main.py]: I tried to add ../main/controllers.py with only code from above picture, and replaced main.py-code with only:

if __name__ == '__main__':
        app.debug = True
        app.run(host = '0.0.0.0', port=5000)

I didnt get it working... could you maybe try to make the change or tell what Im doing wrong :)?

**Pull request merged**